wg-api-expression/README.md

```diff
@@ -0,0 +1,31 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# API Expression Working Group
+
+Improve API expressiveness and support API evolution.
```

Comment:
Link to or copy in the relevant data from @lavalamp's [API Expression proposal](https://docs.google.com/document/d/1XYbQXfge2qKM9psksfC5XZnW8hybtLqL1EcJLU4JwKg) here?

Reply:
Best to copy-paste, for easier access.

Reply:
I tried to copy-paste.
The generation tool handles the mission statement as FoldedStyle which breaks all formating and lists etc.
If we really want the full document in there, i got a change that will switch the generation tool to FlowStyle and we can get the full document in here.